src/util/strencodings.cpp

```diff
@@ -188,9 +188,9 @@ std::vector<unsigned char> DecodeBase64(const char* p, bool* pfInvalid)
     return ret;
 }
 
-std::string DecodeBase64(const std::string& str)
+std::string DecodeBase64(const std::string& str, bool* pfInvalid)
```

Comment:
nit: variable name should be snake_case. See doc/developer-notes.md for naming convention.

Reply:
This was copied from surrounding context (it just wraps the c_str version of the same function, which has the same parameter named the same thing, and I'm propagating it outwards to the wrapper.)

Reply:
I will fix the argument name, but otherwise leave things alone unless you would strongly prefer the more invasive changes.

Reply:
Yeah, let's not do too much in this PR.